cli/command/service/scale.go

```diff
@@ -56,20 +69,31 @@ func runScale(dockerCli *command.DockerCli, args []string) error {
 			continue
 		}
 
-		if err := runServiceScale(dockerCli, serviceID, scale); err != nil {
+		if err := runServiceScale(ctx, dockerCli, serviceID, scale); err != nil {
+			errs = append(errs, fmt.Sprintf("%s: %v", serviceID, err))
+		}
+
+		if options.detach {
+			continue
+		}
+
+		if err := waitOnService(ctx, dockerCli, serviceID, false); err != nil {
```

Comment:
This will change the behavior to wait for the first service to scale before scaling the next one. Previously, they would all be scaled at the same time. I wonder if it's better to continue scaling them simultaneously, and show the progress for all services at the same time. On the other hand, this is a lot more work, and I'm not sure enough people try to scale multiple services at once for this to matter. Any thoughts?